docs/installation.rst

```diff
@@ -0,0 +1,188 @@
+Installation
+===============
+
+Local Kubernetes installation
+-----------------------------
+
+Requirements
+````````````
+In order to create a local REANA cluster with Kubernetes as a backend
+the following requirements must be met:
+
+- `VirtualBox`
+- `VT-x/AMD-v` virtualization must be enabled in BIOS
+
+Quickstart
+``````````
+
+We start by creating a fresh new Python virtual environment that will handle our `REANA` cluster instantiation.
+
+.. code-block:: console
+
+   $ mkvirtualenv reana
+
+We update ``pip`` and ``setuptools``
+
+.. code-block:: console
+
+   $ pip install --upgrade pip setuptools
+
+Next we install ``reana`` package
+
+.. code-block:: console
+
+   $ git clone https://github.com/reanahub/reana
+   $ cd reana
+   $ pip install -e .[all]
+
+Now, for instantiating a local REANA `minikube` based cluster we should run
+
+.. code-block:: console
+
+   $ reana install-minikube
+
+which will do the following:
+
+.. literalinclude:: ../scripts/reana
+    :language: sh
+    :lines: 23-89
+
+Once the script finishes, we should check ``kubectl get pods`` until we have an output as follows:
+
+.. code-block:: console
+
+   $ kubectl get pods
+   job-controller-1390584237-0lt03        1/1       Running            0          1m
+   message-broker-1410199975-7c5v7        1/1       Running            0          1m
+   storage-admin                          1/1       Running            0          1m
+   workflow-controller-2689978795-5kzgt   1/1       Running            0          1m
+   workflow-monitor-1639319062-q7v8z      1/1       Running            0          1m
+   yadage-alice-worker-1624764635-x8z71   1/1       Running            0          1m
+   yadage-atlas-worker-2909073811-t9qv3   1/1       Running            0          1m
+   yadage-cms-worker-209120003-js5cv      1/1       Running            0          1m
+   yadage-lhcb-worker-4061719987-6gpbn    1/1       Running            0          1m
+   zeromq-msg-proxy-1617754619-68p7v      1/1       Running            0          1m
+
+Now we will be able to check whether the nodes are actually working. Firstly, we get the VM's ip:
+
+.. code-block:: console
+
+   $ export MINIKUBE_IP=$(minikube ip)
+
+Secondly, we get the current instances ports:
+
+.. code-block:: console
+
+   $ export WORKFLOW_CONTROLLER_PORT=$(kubectl describe service workflow-controller | grep 'NodePort:' | cut -f 4 | sed -e "s@/TCP@@")
```

Comment:
For these and other environment variables, it would be good to use the common prefix `REANA_...` so that we don't create a mess in people's environments...